README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# hw01
+## About me
+## My name is Jared and I'm from Colorado. 
+### My Hobbies
+* I play **guitar** and love Rock and Alternative music
+* I love science fiction and fantasy stories
+  + My favorite book is *Hitchhiker's Guide to the Galaxy*
+* I am a big fan of movies and video games
+  + Some of my favorite movies are *The Matrix* and *Monty Python and the Holy Grail*
+* I act and have been doing mainly Shakespeare plays lately
+
```

Comment:
The division of information about yourself into the first headline, then the list, and then the short paragraph seems kind of arbitrary. I would appreciate a more consistent guiding theme for deciding which piece of information goes where.